### ISSUE TITLE
Passar o setor changelog do README para o CHANGELOG.

### ISSUE THREAD

**Issue by george-veras** [B2W-BIT/asgard-ui#19]

**Comment by apfzvd** [comment]
A ideia é deixar um link para o CHANGELOG no README e migrar todas as informações para ele?

**Comment by george-veras** [comment]
> A ideia é deixar um link para o CHANGELOG no README e migrar todas as informações para ele?

Sim, na verdade nem havia pensado no link, mas seria bom ter o link mesmo, vai acrescentar navegabilidade na documentação o que é ótimo.
